services/frontend/.env

```diff
@@ -1,6 +1,6 @@
-REACT_APP_GRAPHQL_HTTP_URL=https://graphql-jungle.eosrate.io/v1alpha1/graphql
+REACT_APP_GRAPHQL_HTTP_URL=http://localhost:8088/v1/graphql
```

Comment:
@xavier506 it is necessary to let the previous values, isn't it?
I think I forgot to remove this

Reply:
we should have `.env` files in the gitignore and use github secrets to manage env vars passed along in the github workflows

Reply:
@xavier506 some time ago I added .env files to .gitignore and we had some problems in the pipeline. For now I think it is better to just replace the values with the previous ones